<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateVertex_bt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CreateVertex</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateEdge_bt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CreateEdge</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="from_sb">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="to_sb">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="XML_te">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>461</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>XML code:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Pl&amp;ik</string>
    </property>
    <addaction name="actionZapisz_graf"/>
    <addaction name="actionWczytaj_graf"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdytuj">
    <property name="title">
     <string>Edytu&amp;j</string>
    </property>
    <addaction name="actionDodaj_wezel"/>
    <addaction name="actionUsun_wezel"/>
    <addaction name="actionZmodyfikuj_wezel"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuEdytuj"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionZapisz_graf">
   <property name="text">
    <string>Zapisz graf</string>
   </property>
  </action>
  <action name="actionWczytaj_graf">
   <property name="text">
    <string>Wczytaj graf</string>
   </property>
  </action>
  <action name="actionDodaj_wezel">
   <property name="text">
    <string>Dodaj wezel</string>
   </property>
  </action>
  <action name="actionUsun_wezel">
   <property name="text">
    <string>Usun wezel</string>
   </property>
  </action>
  <action name="actionZmodyfikuj_wezel">
   <property name="text">
    <string>Zmodyfikuj wezel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
